pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1191,23 +1191,23 @@ func runCDCInitialScanRollingRestart(
 	}
 }
 
+type FineGrainedCheckpointingParams struct {
```

Comment:
nit: we don't need to export this struct since we're not using it outside of this package

(in go, all capitalized structs/functions are exported)